src/ol/style/circle.js

```diff
@@ -107,7 +113,7 @@ ol.style.Circle = function(opt_options) {
     opacity: 1,
     rotateWithView: false,
     rotation: 0,
-    scale: 1,
+    scale: (1 / this.pixelRatio_),
```

Comment:
The `#render()` call above should not be necessary, and I also see now that at this call site `this.pixelRatio_` will always be `1`.